source/develop/release-management/features/metrics/setting-up-viaq-logging.md

```diff
@@ -88,11 +88,18 @@ configured to use persistence.
   partition, you are strongly recommended to use a partition other than root
   `/` to avoid filling up the root partition.
 - Find a partition that can easily accomodate many GB of storage.
-- Create the directory e.g. `mkdir -p /var/lib/elasticsearch`
+- Create the directory
+  
+      # mkdir -p /var/lib/elasticsearch
+  
 - Change the group ownership to the value of your
   `openshift_logging_elasticsearch_storage_group` parameter (default `65534`)
-  e.g. `chgrp 65534 /var/lib/elasticsearch`
-- make this directory writable by the group `chmod -R g+w /var/lib/elasticsearch`
+  
+      # chgrp 65534 /var/lib/elasticsearch`
```

Comment:
I think you need a blank line after this line

Reply:
Done. Thanks.

Reply:
@sradco I think you are missing the headers, like these for example:
 ---
 title: oVirt Metrics Store - Installation Guide
 category: feature
 authors: sradco
 feature_name: oVirt Metrics Store Installation
 feature_modules: engine
 feature_status: In Development
---

Please add them, and we will see if the CI pass